akka-cluster-tools/src/main/scala/akka/cluster/client/ClusterClient.scala

```diff
@@ -171,6 +174,80 @@ final class ClusterClientSettings(
       heartbeatInterval, acceptableHeartbeatPause, bufferSize, reconnectTimeout)
 }
 
+/**
+ * Declares a super type for all events emitted by the `ClusterClient`
+ * in relation to contact points being added or removed.
+ */
+sealed trait ContactPointChange {
+  val contactPoint: ActorPath
+}
```

Comment:
Do however you feel with this but I think this supertype can be removed. It's not like you could ever deal with a `ContactPointChange` and it's `contactPoint` without matching on the concrete subtypes as they have very different meanings. (This is highlighted by not using it in any of the tests)